src/test/kotlin/uk/gov/justice/digital/assessments/services/AssessmentServiceTest.kt

```diff
@@ -23,7 +24,9 @@ class AssessmentServiceTest {
 
     @Test
     fun `should save new assessment`() {
+        every { assessmentRepository.findAll(any<Example<AssessmentEntity>>()) } returns emptyList<AssessmentEntity>()
         every { assessmentRepository.save(any<AssessmentEntity>()) } returns AssessmentEntity(assessmentId = 1)
+
         val assessmentDto = assessmentsService.createNewAssessment("SupervisionId")
         verify(exactly = 1) { assessmentRepository.save(any<AssessmentEntity>()) }
```

Comment:
It would be nice to have a unit test in the service test to check the functionality for an existing assessment.

Reply:
Given the controller tests exercising the same code, adding a test in the service for the same operation (especially with the respository mocked out) feels redundant, at least with the very small amount of code we have at the moment.